### PR TITLE
Add boost

### PR DESCRIPTION
**Pull request by rleigh-dundee**
Add boost alongside all the other needed libraries.

### CODE REVIEW

**Comment by sbesson** [comment]
Will test on cowfish this afternoon.

File: ansible/roles/omero-build/tasks/main.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@
     - libtiff-devel
     - xerces-c-devel
     - zlib-devel
+    - boost-devel
```
Comment (manics) [inline ansible/roles/omero-build/tasks/main.yml]: very nitpicky: alphabetical order

**Comment by rleigh-dundee** [comment]
@manics Fixed.

**Comment by sbesson** [comment]
Tested on cowfish

```
Perform task: omero-build | system packages | install build libraries (y/n/c):  *** 
changed: [cowfish.openmicroscopy.org] => (item=boost-devel,libjpeg-turbo-devel,libpng-devel,libtiff-devel,xerces-c-devel,zlib-devel) => {"changed": true, "item": "boost-devel,libjpeg-turbo-devel,libpng-devel,libtiff-devel,xerces-c-devel,zlib-devel", "msg": "", "rc": 0, "results": ["All packages providing libjpeg-turbo-devel are up to date", "All packages providing libpng-devel are up to date", "All packages providing libtiff-devel are up to date", "All packages providing xerces-c-devel are up to date", "All packages providing zlib-devel are up to date", "Loaded plugins: fastestmirror, langpacks, rhnplugin\nThis system is receiving updates from RHN Classic or Red Hat Satellite.\nLoading mirror speeds from cached hostfile\n * base: mirror.bytemark.co.uk\n * epel: mirror.bytemark.co.uk\n * extras: mirrors.ukfast.co.uk\n * updates: mirror.bytemark.co.uk\nResolving Dependencies\n--> Running transaction check\n---> Package boost-devel.x86_64 0:1.53.0-23.el7 will be installed\n--> Processing Dependency: boost = 1.53.0-23.el7 for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_wserialization.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_wserialization-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_wave.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_wave-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_unit_test_framework.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_unit_test_framework-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_timer.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_timer-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_signals.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_signals-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_serialization.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_serialization-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_regex.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_regex-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_random.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_random-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_python.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_python-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_program_options.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_program_options-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_prg_exec_monitor.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_prg_exec_monitor-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_tr1l.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_tr1l-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_tr1f.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_tr1f-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_tr1.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_tr1-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_c99l.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_c99l-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_c99f.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_c99f-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_c99.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_math_c99-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_locale-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_iostreams.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_iostreams-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_graph.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_graph-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_filesystem.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_filesystem-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_date_time.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_date_time-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_context.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_context-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_chrono.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_chrono-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Processing Dependency: libboost_atomic-mt.so.1.53.0()(64bit) for package: boost-devel-1.53.0-23.el7.x86_64\n--> Running transaction check\n---> Package boost.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-atomic.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-chrono.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-context.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-date-time.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-filesystem.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-graph.x86_64 0:1.53.0-23.el7 will be installed\n--> Processing Dependency: libicuuc.so.50()(64bit) for package: boost-graph-1.53.0-23.el7.x86_64\n--> Processing Dependency: libicui18n.so.50()(64bit) for package: boost-graph-1.53.0-23.el7.x86_64\n--> Processing Dependency: libicudata.so.50()(64bit) for package: boost-graph-1.53.0-23.el7.x86_64\n---> Package boost-iostreams.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-locale.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-math.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-program-options.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-python.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-random.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-regex.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-serialization.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-signals.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-test.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-timer.x86_64 0:1.53.0-23.el7 will be installed\n---> Package boost-wave.x86_64 0:1.53.0-23.el7 will be installed\n--> Running transaction check\n---> Package libicu.x86_64 0:50.1.2-11.el7 will be installed\n--> Finished Dependency Resolution\n\nDependencies Resolved\n\n================================================================================\n Package                     Arch         Version              Repository  Size\n================================================================================\nInstalling:\n boost-devel                 x86_64       1.53.0-23.el7        base       7.0 M\nInstalling for dependencies:\n boost                       x86_64       1.53.0-23.el7        base        32 k\n boost-atomic                x86_64       1.53.0-23.el7        base        34 k\n boost-chrono                x86_64       1.53.0-23.el7        base        43 k\n boost-context               x86_64       1.53.0-23.el7        base        35 k\n boost-date-time             x86_64       1.53.0-23.el7        base        50 k\n boost-filesystem            x86_64       1.53.0-23.el7        base        67 k\n boost-graph                 x86_64       1.53.0-23.el7        base       135 k\n boost-iostreams             x86_64       1.53.0-23.el7        base        60 k\n boost-locale                x86_64       1.53.0-23.el7        base       250 k\n boost-math                  x86_64       1.53.0-23.el7        base       332 k\n boost-program-options       x86_64       1.53.0-23.el7        base       155 k\n boost-python                x86_64       1.53.0-23.el7        base       131 k\n boost-random                x86_64       1.53.0-23.el7        base        38 k\n boost-regex                 x86_64       1.53.0-23.el7        base       294 k\n boost-serialization         x86_64       1.53.0-23.el7        base       169 k\n boost-signals               x86_64       1.53.0-23.el7        base        59 k\n boost-test                  x86_64       1.53.0-23.el7        base       225 k\n boost-timer                 x86_64       1.53.0-23.el7        base        42 k\n boost-wave                  x86_64       1.53.0-23.el7        base       211 k\n libicu                      x86_64       50.1.2-11.el7        base       6.9 M\n\nTransaction Summary\n================================================================================\nInstall  1 Package (+20 Dependent packages)\n\nTotal download size: 16 M\nInstalled size: 122 M\nDownloading packages:\n--------------------------------------------------------------------------------\nTotal                                               12 MB/s |  16 MB  00:01     \nRunning transaction check\nRunning transaction test\nTransaction test succeeded\nRunning transaction\n  Installing : boost-chrono-1.53.0-23.el7.x86_64                           1/21 \n  Installing : libicu-50.1.2-11.el7.x86_64                                 2/21 \n  Installing : boost-regex-1.53.0-23.el7.x86_64                            3/21 \n  Installing : boost-date-time-1.53.0-23.el7.x86_64                        4/21 \n  Installing : boost-filesystem-1.53.0-23.el7.x86_64                       5/21 \n  Installing : boost-wave-1.53.0-23.el7.x86_64                             6/21 \n  Installing : boost-graph-1.53.0-23.el7.x86_64                            7/21 \n  Installing : boost-locale-1.53.0-23.el7.x86_64                           8/21 \n  Installing : boost-timer-1.53.0-23.el7.x86_64                            9/21 \n  Installing : boost-signals-1.53.0-23.el7.x86_64                         10/21 \n  Installing : boost-context-1.53.0-23.el7.x86_64                         11/21 \n  Installing : boost-iostreams-1.53.0-23.el7.x86_64                       12/21 \n  Installing : boost-atomic-1.53.0-23.el7.x86_64                          13/21 \n  Installing : boost-program-options-1.53.0-23.el7.x86_64                 14/21 \n  Installing : boost-random-1.53.0-23.el7.x86_64                          15/21 \n  Installing : boost-serialization-1.53.0-23.el7.x86_64                   16/21 \n  Installing : boost-test-1.53.0-23.el7.x86_64                            17/21 \n  Installing : boost-math-1.53.0-23.el7.x86_64                            18/21 \n  Installing : boost-python-1.53.0-23.el7.x86_64                          19/21 \n  Installing : boost-1.53.0-23.el7.x86_64                                 20/21 \n  Installing : boost-devel-1.53.0-23.el7.x86_64                           21/21 \n  Verifying  : boost-filesystem-1.53.0-23.el7.x86_64                       1/21 \n  Verifying  : boost-python-1.53.0-23.el7.x86_64                           2/21 \n  Verifying  : boost-devel-1.53.0-23.el7.x86_64                            3/21 \n  Verifying  : boost-timer-1.53.0-23.el7.x86_64                            4/21 \n  Verifying  : boost-regex-1.53.0-23.el7.x86_64                            5/21 \n  Verifying  : boost-date-time-1.53.0-23.el7.x86_64                        6/21 \n  Verifying  : libicu-50.1.2-11.el7.x86_64                                 7/21 \n  Verifying  : boost-wave-1.53.0-23.el7.x86_64                             8/21 \n  Verifying  : boost-math-1.53.0-23.el7.x86_64                             9/21 \n  Verifying  : boost-test-1.53.0-23.el7.x86_64                            10/21 \n  Verifying  : boost-chrono-1.53.0-23.el7.x86_64                          11/21 \n  Verifying  : boost-serialization-1.53.0-23.el7.x86_64                   12/21 \n  Verifying  : boost-locale-1.53.0-23.el7.x86_64                          13/21 \n  Verifying  : boost-random-1.53.0-23.el7.x86_64                          14/21 \n  Verifying  : boost-program-options-1.53.0-23.el7.x86_64                 15/21 \n  Verifying  : boost-atomic-1.53.0-23.el7.x86_64                          16/21 \n  Verifying  : boost-iostreams-1.53.0-23.el7.x86_64                       17/21 \n  Verifying  : boost-1.53.0-23.el7.x86_64                                 18/21 \n  Verifying  : boost-context-1.53.0-23.el7.x86_64                         19/21 \n  Verifying  : boost-graph-1.53.0-23.el7.x86_64                           20/21 \n  Verifying  : boost-signals-1.53.0-23.el7.x86_64                         21/21 \n\nInstalled:\n  boost-devel.x86_64 0:1.53.0-23.el7                                            \n\nDependency Installed:\n  boost.x86_64 0:1.53.0-23.el7                                                  \n  boost-atomic.x86_64 0:1.53.0-23.el7                                           \n  boost-chrono.x86_64 0:1.53.0-23.el7                                           \n  boost-context.x86_64 0:1.53.0-23.el7                                          \n  boost-date-time.x86_64 0:1.53.0-23.el7                                        \n  boost-filesystem.x86_64 0:1.53.0-23.el7                                       \n  boost-graph.x86_64 0:1.53.0-23.el7                                            \n  boost-iostreams.x86_64 0:1.53.0-23.el7                                        \n  boost-locale.x86_64 0:1.53.0-23.el7                                           \n  boost-math.x86_64 0:1.53.0-23.el7                                             \n  boost-program-options.x86_64 0:1.53.0-23.el7                                  \n  boost-python.x86_64 0:1.53.0-23.el7                                           \n  boost-random.x86_64 0:1.53.0-23.el7                                           \n  boost-regex.x86_64 0:1.53.0-23.el7                                            \n  boost-serialization.x86_64 0:1.53.0-23.el7                                    \n  boost-signals.x86_64 0:1.53.0-23.el7                                          \n  boost-test.x86_64 0:1.53.0-23.el7                                             \n  boost-timer.x86_64 0:1.53.0-23.el7                                            \n  boost-wave.x86_64 0:1.53.0-23.el7                                             \n  libicu.x86_64 0:50.1.2-11.el7                                                 \n\nComplete!\n"]}
````

Barring any veto, will merge once green.

**Comment by rleigh-dundee** [comment]
Looks good on cowfish; thanks.

**Pull request merged**